Add some pricing page styles to styleguide - no docs yet

louh opened the pull request:
These are styles for the sub-page hero banner and the primary call-to-action banners used on the pricing page.

louh committed:
Add some pricing page styles to styleguide - no docs yet

hanbyul-here commented:
I love Robin :bat: I tried to get the right spacing in this pr: https://github.com/mapzen/styleguide/pull/434/files It's fine if you want to use hard-coded number for now, but eventually it would be better to use vars?

louh commented:
Variables would be 100% better. I'll add those.

louh committed:
Redefine secondary heroes in terms of revised homepage hero; use padding variables.

louh committed:
Revise .cta-primary padding to use variables

louh committed:
Move section spacing variables to global variables so other components can use it

louh commented:
@hanbyul-here I've updated this so that Robin uses your revised markup for Batman (it's a huge improvement!) and redefined the styles accordingly. It's also now using the section variables. Since I'm proposing the call-to-action banners as a generic component, rather than "blog"-specific, I've also moved the `$section-padding` variables into the global variables file.

hanbyul-here commented on src/stylesheets/project_specific/blog/blog.scss:
```diff
@@ -1,6 +1,4 @@
 $section-height: 550px;
```
It will be probably better to rename this var to specific thing (ex.map gallery height) and make it to local to that file. I will do that in separate PR

louh committed:
Merge branch 'master' into lou/pricing-styles
